pkg/controller/subscription/subscription_controller.go

```diff
@@ -139,7 +140,7 @@ type ReconcileSubscription struct {
 func (r *ReconcileSubscription) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	// skip any Subscriptions that are not integreatly operator
 	if !r.shouldReconcileSubscription(request) {
-		logrus.Infof("not our subscription: %+v, %s", request, r.operatorNamespace)
+		log.Infof("Not our subscription", l.Fields{"request": request, "opNS": r.operatorNamespace})
```

Comment:
Another question, why do we sometimes, for example in RHMI config controller have the logger returned in `newReconciller` and use the `r.log.xxxx` and as in the example above we use it directly via `log.xxxx` - I am probably missing something and wonder if you could help me understand the difference?

Reply:
Thats a bit inconsistent. I've changed the RHMI config controller to just log.

In some cases, the product reconcilers for example, we use the logger from the installation controller so it makes sense to add it to the reconciler. But not for RHMI config controller for you're right to point that out.